dev-requirements.txt

```diff
@@ -19,3 +19,4 @@ asgiref==3.7.2
 psutil==5.9.6
 GitPython==3.1.41
 flaky==3.7.0
+pre-commit==3.7.0
```

Comment:
I realize this change has been merged, but I bumped into an issue where when I tried to run eachdist develop in a Python 3.8 venv, it failed because Python 3.8 (which this repo currently supports) is compatible only up to pre-commit  [3.5.0](https://pypi.org/project/pre-commit/3.5.0/). Pre-commit [3.6.0](https://pypi.org/project/pre-commit/3.6.0/) requires Python 3.9+.

Can the requirement on this line be relaxed to pre-commit 3.5.0?

Reply:
Opened https://github.com/open-telemetry/opentelemetry-python/pull/3910